Turf/Turf.swift

```diff
@@ -4,10 +4,9 @@ public typealias LocationRadians = Double
 public typealias RadianDistance = Double
 public typealias RadianDirection = Double
 
-struct Constants {
-    static let metersPerRadian = 6_373_000.0
-    static let equatorialRadius:Double = 6378137
-}
+
+let metersPerRadian: CLLocationDistance = 6_373_000.0
```

Comment:
Apparently this value of 6&nbsp;373&nbsp;000&nbsp;m came from [an old version of turf-distance](https://github.com/Turfjs/turf-distance/blob/cea4c6cbec582bff8880193a3c2cda38ae731967/index.js#L78) that I originally ported to an internal Swift application, before it got copied into another internal Swift application, then copied into the navigation SDK, then moved here. It’s close to the value of 6&nbsp;372&nbsp;797.560&nbsp;856 that [Osmium](https://github.com/Project-OSRM/osrm-backend/blob/bb1f4a025a3cefd3598a38b9d3e55485d1080ec5/third_party/libosmium/include/osmium/geom/haversine.hpp#L58-L59) describes as “Earth’s quadratic mean radius for WGS84”.

These days, Turf.js uses [a spherical approximation](https://github.com/Turfjs/turf/blob/a94151418cb969868fdb42955a19a133512da0fd/packages/turf-helpers/index.js#L4) of 6&nbsp;671&nbsp;008.8&nbsp;m for its radian-to-meter conversion and Haversine formula. The Haversine formula was always meant to be used with a spherical meters-per-radian value. We should probably change this value to match Turf.js. 🙀

/ref Turfjs/turf#978 Turfjs/turf#1012 Turfjs/turf#1176
/cc @frederoni @bsudekum

Reply:
Since the polygon area calculation doesn’t depend on `metersPerRadian`, we can treat this change as tail work: #26.

Reply:
If we're going to change `metersPerRadian ` to `6 671 008.8 m`, should this happen in a different PR since it would globally impact this library? Or are you ok with me making the change here?

Reply:
Yeah, let’s do it in a separate PR for #26, so that any fallout is easier to track down.

In the meantime, can you add some comments explaining why these two constants differ?